### PR TITLE
ENH Add CUDA 11.1 support

### PR DESCRIPTION
**Pull request by mike-wendt**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Follows #26 with a new PR that uses the updated conda build layout

### CODE REVIEW

**Comment by mike-wendt** [comment]
This will still have the same `cupti` issues as #26 and I need help to get this solved, this is to start the process and solicit feedback to get it over the line

**Comment by kkraus14** [comment]
> This will still have the same `cupti` issues as #26 and I need help to get this solved, this is to start the process and solicit feedback to get it over the line

Given this is a separate issue is there any reason we shouldn't move forward with this and then address cupti in a follow up?

**Review by kkraus14 — DISMISSED** [review]

**Comment by isuruf** [comment]
> Given this is a separate issue is there any reason we shouldn't move forward with this and then address cupti in a follow up?

Sure. That's only because this is not publishing to `main`. In all other feedstocks, things like this would be blockers as merging would publish the package to `main`. Please create a new issue before merging this PR. (I doubt this issue is still there as those libraries were dropped)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -171,7 +193,7 @@ source:
     md5: {{ cudavars[major_minor]["checksums"]["win"] }}  # [win]
 
 build:
-  number: 4
+  number: 0
```
Comment (kkraus14) [inline recipe/meta.yaml:196]: Should we be setting this back to zero given CI fans out the builds for all of the CUDA versions anyway?

Comment (mike-wendt) [reply recipe/meta.yaml:196]: guess not, I'll revert

Comment (mike-wendt) [reply recipe/meta.yaml]: @kkraus14 resolved

File: recipe/meta.yaml [hunk 2]
```diff
@@ -171,7 +193,7 @@ source:
     md5: {{ cudavars[major_minor]["checksums"]["win"] }}  # [win]
 
 build:
-  number: 4
+  number: 5
```
Comment (isuruf) [inline recipe/meta.yaml:196]: Keep this `4` if there's no change to the other packages

**Review by kkraus14 — APPROVED** [review]

**Comment by kkraus14** [comment]
Merging, thanks @mike-wendt!

**Comment by jakirkham** [comment]
Thanks Mike! Also thanks Keith and Isuru for the reviews! 😄

**Pull request merged**